src/app/containers/Include/index.test.jsx

```diff
@@ -195,4 +195,26 @@ describe('IncludeContainer', () => {
         '/include/newsspec/21841-green-diet/gahuza/app?responsive=true&newsapps=true&app-image=https://news.files.bbci.co.uk/vj/live/idt-images/image-slider-asdf/app_launcher_ws_640_7ania.png&app-clickable=true&amp-clickable=true&amp-image-height=360&amp-image-width=640&amp-image=https://news.files.bbci.co.uk/vj/live/idt-images/image-slider-asdf/app_launcher_ws_640_7ania.png',
     });
   });
+
+  it('should render a fallback for VJs on an Amp page when showEmbed is present', () => {
```

Comment:
A small thing, but if we're referencing a variable in the title, it should probably be `showEmbedError`.